README.md

```diff
@@ -30,7 +30,7 @@ You will find some  more general considerations about the LWM2M Mapping we are u
 In order to assign the proper configuration for each type of device, a mechanism to assign types to new arriving devices should be used. This mechanism is based on Prefixes for the registrarion URL. For each type configured in the `lwm2m` configuration section in the `config.js` config file, a url prefix has to be defined. Whenever a registration arrives to an url with that prefix, the device will be assigned the corresponding type.
 
 ## Mappings
-One of the features to provide through the IoT Agent is the mapping between the protocol specific names of the South Bound to the application-specific names in the North Bound. In the case of Lightweight M2M, this means to map two things:
+One of the features to provide through the IoT Agent is the mapping between the protocol specific names of the South Port traffic to the application-specific names in the North Port. In the case of Lightweight M2M, this means to map two things:
```

Comment:
In line 52, Nortbound and Southbound also appear.

Reply:
I think that the unaltered text is correct on line 52. - i.e. There are no security mechanisms when traffic is flowing from NGSI->IoT Agent->Device  **Southbound** or Device->IoT Agent-> NSGI **Northbound**

Reply:
@dcalvoalonso - I think it should be updated.  Fixed  b1deab1 (for consistency)